doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -796,8 +585,8 @@ correctly. If there is a problem with these or other variables (e.g., file paths
 .. code-block:: console
    :emphasize-lines: 5,6
 
-   + echo 'Machine: ' hera.intel '    Account: ' nems
-   Machine:  hera.intel     Account:  nems
+   + echo 'Machine: ' ursa.intel '    Account: ' nems
```

Comment:
In L777 and 785 above, the `-a <account>` argument should be added because that is a mandatory argument for running the script.